src/test/java/com/conveyal/datatools/manager/jobs/DeployJobTest.java

```diff
@@ -30,6 +30,8 @@
  *
  * Note: these tests require credentials on the IBI AWS account, which is why the class is tagged with Ignore (so that it is
  * not run with the rest of the test suite)
+ *
+ * FIXME: Does this need to extend UnitTest?
```

Comment:
Yes, every test class that is not part of the e2e tests should extend the `com.conveyal.datatools.UnitTest` class. The `com.conveyal.datatools.UnitTest` class has some logic that will skip the tests if only e2e tests should be ran.